html+css

Liza-Aleks1108 opened the pull request:
1. Проблема з фото lens. Намагаюсь вирішити, але поки не вийшло. Фото не так центруеться. З камерою все good.

Liza-Aleks1108 committed:
html+css

1. Проблема з фото lens. Намагаюсь вирішити, але поки не вийшло. Фото не так центруеться. З камерою все good.

Serhii-Ivanchenko commented on src/partials/sign-up.html:
```diff
@@ -0,0 +1,89 @@
+<div class="container">
```
div.container треба вкласти в section (для секції оформлення ніякого не треба задавати окрім того, що є в базовому стилі), і тоді тобі пустий div не треба буде, а до існуючого div .container додай якийсь клас додатковий, якщо потрібно додаткове оформлення.

Serhii-Ivanchenko commented on src/css/sign-up.css:
```diff
@@ -0,0 +1,178 @@
+.signup {
+  width: 335px;
+  line-height: 1.29;
```
перенеси line-height в стилі для конкретних текстів

Serhii-Ivanchenko commented on src/css/sign-up.css:
```diff
@@ -0,0 +1,178 @@
+.signup {
+  width: 335px;
```
ширину з секції прибери, вона у нас формується контейнером, а ширину форми для десктопу задаємо для .signup--wrapper

Serhii-Ivanchenko commented on src/css/sign-up.css:
```diff
@@ -0,0 +1,178 @@
+  border: 1px solid rgba(255, 255, 255, 0.50);
+}
+
+.textarea {
```
сюди додай ще :
display: block;
    resize: none;

Serhii-Ivanchenko commented on src/css/sign-up.css:
```diff
@@ -0,0 +1,178 @@
+  margin-bottom: 20px;
+}
+
+.signup-input {
```
сюди додай ще:
outline: transparent;

Serhii-Ivanchenko reviewed (changes requested):
виправ поки це, далі побачимо)

Liza-Aleks1108 committed:
Merge branch 'main' into origin/sign-up

Liza-Aleks1108 committed:
sign up change

1.Виправила в медіа запиті min-width: 1158px на min-width: 1280px
2.Зробила з signup:  display: flex;
3.Додала display: block; для .signup-input та resize: none; для .signup-textarea
4.Додала outline: transparent; для .signup-wrapper
5.Виправила фото об'єктива. Але поки проблема з тінню.

Liza-Aleks1108 committed:
signup shadow

Зробила ту тінь на картинку

Liza-Aleks1108 committed:
Merge branch 'main' into origin/sign-up

Liza-Aleks1108 committed:
Change signup-button

Прибрала padding в signup-button

Liza-Aleks1108 committed:
Сhange section and div
